mount.go

```diff
@@ -215,6 +230,29 @@ func ephemeralStorageHandler(storage pb.Storage, s *sandbox) (string, error) {
 	return "", nil
 }
 
+func localStorageHandler(storage pb.Storage, s *sandbox) (string, error) {
+	s.Lock()
+	defer s.Unlock()
+	newStorage := s.setSandboxStorage(storage.MountPoint)
+	if newStorage {
+
+		// Extract and parse the mode out of the storage options.
+		// Default to os.ModePerm.
+		opts := parseOptions(storage.Options)
+		mode := os.ModePerm
+		if val, ok := opts["mode"]; ok {
```

Comment:
So we pass dir `mode` here. Please add the corresponding part in kata-containers/runtime#1485

Reply:
Thanks for the recommendation, I've updated the runtime PR with the mode.